lib/posix/Kconfig

```diff
@@ -182,3 +182,26 @@ config FNMATCH
 	help
 	  Match filenames using the the fnmatch function. For example, the pattern
 	  "*.c" matches the filename "hello.c".
+
+config POSIX_UNAME
+	bool "Support for uname"
+	default y if POSIX_API
+	help
+	  The uname() function shall store information identifying the current
+	  system in the structure pointed to by name.
+
+if POSIX_UNAME
+config POSIX_UNAME_VERSION_LEN
+	int "Buffer size for version"
+	default 60
+	help
+	  Size of the version char array in the utsname struct.
+
+config POSIX_UNAME_NODENAME_LEN
```

Comment:
I looked at the Linux version of this code and it defines the maximum string length and then adds one when declaring the array to account for the null terminator. Seems like a good idea. Would mean changing the doc strings here to talk about 'maximum length of the node name'. (the doc string here was also copied from the version one above).